HarfangLab/harfanglab/tests/powershell.json

```diff
@@ -0,0 +1,49 @@
+{
+  "input": {
+    "message": "{\"groups\": [{\"id\": \"14989c86-9f2f-4d8d-92b5-1210034ca640\", \"name\": \"SERVERS\"}], \"script_block\": \"$srv=Get-Service 'CARLTC85-CS02' -ErrorAction SilentlyContinue;\\t\\t\\t\\t\\tif ($srv) { Write-Output $srv.status; exit 0 } else { exit 1 }\", \"process_image_path\": \"C:\\\\Windows\\\\System32\\\\WindowsPowerShell\\\\v1.0\\\\powershell.exe\", \"sha256\": \"cebeed9684d8b8cbd821db554e5bb4937ffa1d32f2ef57481866c5f0b09838b0\", \"@event_create_date\": \"2024-08-02T09:03:13.833000Z\", \"@version\": \"1\", \"log_type\": \"powershell\", \"@timestamp\": \"2024-08-02T08:56:47.681487513Z\", \"pid\": 26748, \"tenant\": \"e13f90f553ce3a9e\", \"script_path\": \"\", \"agent\": {\"osproducttype\": \"Windows Server 2016 Datacenter\", \"version\": \"3.8.16\", \"additional_info\": {}, \"ostype\": \"windows\", \"osversion\": \"10.0.14393\", \"hostname\": \"srv-gmao-prod\", \"distroid\": null, \"dnsdomainname\": \"cmnshipyard.local\", \"agentid\": \"9f298e79-f284-4276-9176-83a02c40c8f0\", \"domain\": null, \"domainname\": \"CMNSHIPYARD\"}, \"incomplete\": false, \"process_unique_id\": \"de5434d5-f284-4276-7c68-00312b6f2996\"}"
+  },
+  "expected": {
+    "message": "{\"groups\": [{\"id\": \"14989c86-9f2f-4d8d-92b5-1210034ca640\", \"name\": \"SERVERS\"}], \"script_block\": \"$srv=Get-Service 'CARLTC85-CS02' -ErrorAction SilentlyContinue;\\t\\t\\t\\t\\tif ($srv) { Write-Output $srv.status; exit 0 } else { exit 1 }\", \"process_image_path\": \"C:\\\\Windows\\\\System32\\\\WindowsPowerShell\\\\v1.0\\\\powershell.exe\", \"sha256\": \"cebeed9684d8b8cbd821db554e5bb4937ffa1d32f2ef57481866c5f0b09838b0\", \"@event_create_date\": \"2024-08-02T09:03:13.833000Z\", \"@version\": \"1\", \"log_type\": \"powershell\", \"@timestamp\": \"2024-08-02T08:56:47.681487513Z\", \"pid\": 26748, \"tenant\": \"e13f90f553ce3a9e\", \"script_path\": \"\", \"agent\": {\"osproducttype\": \"Windows Server 2016 Datacenter\", \"version\": \"3.8.16\", \"additional_info\": {}, \"ostype\": \"windows\", \"osversion\": \"10.0.14393\", \"hostname\": \"srv-gmao-prod\", \"distroid\": null, \"dnsdomainname\": \"cmnshipyard.local\", \"agentid\": \"9f298e79-f284-4276-9176-83a02c40c8f0\", \"domain\": null, \"domainname\": \"CMNSHIPYARD\"}, \"incomplete\": false, \"process_unique_id\": \"de5434d5-f284-4276-7c68-00312b6f2996\"}",
```

Comment:
Can you extract `process.image.path` into ECS field `process.executable`?
Can you also add a smart-description for this event?